packages/twenty-front/src/pages/settings/SettingsProfile.tsx

```diff
@@ -45,6 +60,26 @@ export const SettingsProfile = () => {
           />
           <EmailField />
         </Section>
+        {!!currentWorkspace?.twoFactorAuthenticationPolicy &&
+          isTwoFactorAuthenticationEnabled === true && (
```

Comment:
It seems like because it is  misnamed this introduced a confusion.
There are 2 distinct things:
- Is the 2 factor feature enabled at a server level (not sure why we'd want to disable it so I would have advocated to remove it to lower complexity
- Is 2 factor mandatory for all members of a given workspace

In that case I think it was the wrong one that was used. Regardless of if my workspace admin enforces mandatory 2FA I should still always be able to have access to the 2FA feature to set it up myself